lib/mod.lua

```diff
@@ -90,9 +112,13 @@ mod.hook.register("script_pre_init", "nbout pre init", function()
         params:add_separator("nbout")
         nb:add_param("nbout_chan_1", "nb midi ch 1")
         nb:add_player_params()
+        channel_is_set_up = true
+        print("nbout_chan_1 is set up")
```

Comment:
Can we have this be silent in normal operation? You can leave the print statement here commented out.

Reply:
Sure.  The print statements are there mostly so that we can visualize where things are going wrong during development and when some script fails to start/can't produce sound with nbout.

This particular print statement is supposed to supply context for the "Refused to ..." error messages, so that we can see when the initialization step actually takes place sometime after the errant note_on was sent.  But I think we probably also see a message about nbout initialization hereabouts anyway, so it shouldn't be too hard to understand what is going on when one of those errors shows up.